uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/federatedlogin/uilocators/GeSsoSPlogin.java

```diff
@@ -0,0 +1,22 @@
+package org.cloudfoundry.identity.uaa.integration.feature.federatedlogin.uilocators;
+
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class GeSsoSPlogin {
```

Comment:
GeSsoSPlogin is a confusing class name. You are not using GE SSO for testing. Please think of a better name.

Reply:
Done